xml/obs_build_config.xml

```diff
@@ -334,10 +334,13 @@ Keep: debbuild</screen>
       OPTIONS parameter depends on the repository type, for rpm-md the known
       options are 'legacy' to create the old rpm-md format, 'deltainfo' or
       'prestodelta' to create delta rpm packages, 'rsyncable' to use rsyncable
-      gzip compression. To split the debug packages in an own published
-      repository the option
+      gzip compression.
+      To split the debug packages in an own published repository the option
       <literal>splitdebug:<replaceable>REPOSITORY_SUFFIX</replaceable></literal>
-      can be used. </para>
+      can be appended, e.g. <emphasis>Repotype: rpm-md splitdebug:-debuginfo</emphasis>
```

Comment:
Please change the <emphasis> to <screen></screen>

Reply:
@dsamx I didn't notice that my comment got escaped by github. So the `<emphasis> to <screen>` was missing. I've fixed the comment now.

Could you please update your commit accordingly?:smile: